lab12.md

```diff
@@ -0,0 +1,38 @@
+# Lab 12 - Advanced feature - Scale idle functions to 0
```

Comment:
I'm going to suggest removing Lab 12 completely. I don't think it's adding value over what is in the official documentation.